libs/judicial-system/consts/src/lib/consts.ts

```diff
@@ -75,7 +75,7 @@ export const ICaseTypes = [
 export const TIME_FORMAT = 'HH:mm'
 
 // Routes
-export const REQUEST_LIST_ROUTE = '/krofur'
+export const CASE_LIST_ROUTE = '/krofur'
```

Comment:
Perhaps consult with the product owners on this to make sure old links can be invalidated.

Reply:
I also raised this question when I was working on https://app.asana.com/0/1199153462262248/1201888481479194/f We decided to not do the URL change. I think it would be a small effort to change this, starting with a simple redirect in the web nextjs server.